Dockerfile

```
@@ -1,10 +1,13 @@
 FROM registry.access.redhat.com/ubi8/nodejs-16:latest AS build
 USER root
-RUN command -v yarn || npm i -g yarn
 
-ADD . /usr/src/app
 WORKDIR /usr/src/app
-RUN yarn install && yarn build
+
+COPY package*.json .
+RUN npm install
+
+ADD . /usr/src/app
```

If the workdir is already defined we don't need to copy the whole path. Also the frontend app should be contained within the `web` folder
```suggestion
ADD web/ web/
```

---

Moved frontend app into /web folder in the latest commit.